From d877bdb5231c7e1055c5ee0767497121016b3e13 Mon Sep 17 00:00:00 2001
From: Ananth Krishna R <ananth.krishna.r@intel.com>
Date: Thu, 5 Mar 2015 01:21:17 +0530
Subject: [PATCH] [VRF]: [TI PMIC] Changes to LDO7 and LDO8 voltage tables

This patch modifies voltage tables of LDOs which power SD Card
to disable all voltages except 1.8V & 3.3V. This limitation can
stabilize SD read/write operation.

Change-Id: I6d76a4c419dd39bdc5e29a82245d61a394ce428f
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-6720
Signed-off-by: Ananth Krishna R <ananth.krishna.r@intel.com>
---
 drivers/regulator/pmic_dollar_cove_ti.c | 20 ++++++--------------
 1 file changed, 6 insertions(+), 14 deletions(-)

diff --git a/drivers/regulator/pmic_dollar_cove_ti.c b/drivers/regulator/pmic_dollar_cove_ti.c
index f7a6b61..48357ae 100644
--- a/drivers/regulator/pmic_dollar_cove_ti.c
+++ b/drivers/regulator/pmic_dollar_cove_ti.c
@@ -121,23 +121,15 @@ static const unsigned int LDO6_table[] = {
 };
 
 static const unsigned int LDO7_table[] = {
-	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3300000, 3250000, 3200000,
-	3150000, 3100000, 3050000, 3000000, 2950000, 2900000, 2850000, 2800000,
-	2750000, 2700000, 2650000, 2600000, 2550000, 2500000, 2450000, 2400000,
-	2350000, 2300000, 2250000, 2200000, 2150000, 2100000, 2050000, 2000000,
-	1950000, 1900000, 1850000, 1800000, 1750000, 1700000, 1650000, 1600000,
-	1550000, 1500000, 1450000, 1400000, 1350000, 1300000, 1250000, 1200000,
-	1150000, 1100000, 1050000, 1000000, 950000, 900000, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3300000, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	1800000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 };
 
 static const unsigned int LDO8_table[] = {
-	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3300000, 3250000, 3200000,
-	3150000, 3100000, 3050000, 3000000, 2950000, 2900000, 2850000, 2800000,
-	2750000, 2700000, 2650000, 2600000, 2550000, 2500000, 2450000, 2400000,
-	2350000, 2300000, 2250000, 2200000, 2150000, 2100000, 2050000, 2000000,
-	1950000, 1900000, 1850000, 1800000, 1750000, 1700000, 1650000, 1600000,
-	1550000, 1500000, 1450000, 1400000, 1350000, 1300000, 1250000, 1200000,
-	1150000, 1100000, 1050000, 1000000, 950000, 900000, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3300000, 0, 0, 0, 0, 0, 0,
+	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+	1800000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 };
 
 static const unsigned int LDO9_table[] = {
-- 
1.9.1

